src/components/partner/partner.service.ts

```diff
@@ -82,6 +83,27 @@ export class PartnerService {
       input.financialReportingTypes,
       input.types
     );
+    const partnerByOrgQ = this.db
+      .query()
+      .match([node('node', 'Organization', { id: input.organizationId })])
+      .match([
+        node('node'),
+        relation('in', '', 'organization', { active: true }),
+        node('partner', 'Partner'),
+      ])
+      .return({
+        partner: [{ id: 'partnerId' }],
+      })
+      .asResult<{
+        partnerId: string;
+      }>();
+    const checkPartner = await partnerByOrgQ.first();
+    if (checkPartner) {
+      throw new DuplicateException(
+        'organization.id',
```

Comment:
The field should match the input object
```suggestion
        'partner.organizationId',
```